core/src/main/resources/lib/layout/main-panel.jelly

```diff
@@ -30,7 +30,11 @@ THE SOFTWARE.
 
   <j:if test="${mode=='main-panel'}">
     <j:if test="${app.isQuietingDown()}">
-      <div id='shutdown-msg'>${%Jenkins is going to shut down}</div>
+      <div id='shutdown-msg'>${%Jenkins is going to shut down}
+        <j:if test="${app.getQuietDownReason() != null}">
+          <br/> ${%Shut down reason}: ${app.getQuietDownReason()}
```

Comment:
Markup formatter would be nice, especially in cases when admins would link to a URL showing planned maintenances or something.

```suggestion
          <br/> ${%Shut down reason}: ${app.markupFormatter.translate(app.getQuietDownReason())}
```

That said, probably better off as a future enhancement, as it would need a preview feature for the shutdown reason text field too.